core/state/statedb.go

```diff
@@ -192,7 +192,7 @@ func (s *StateDB) NewAccessWitness() *AccessWitness {
 
 func (s *StateDB) Witness() *AccessWitness {
 	if s.witness == nil {
-		s.witness = s.NewAccessWitness()
+		panic("witness wasn't initialized")
```

Comment:
I had the feeling this removed logic was intentionally trying to lazily initialize in some cases.

Thing is that if we remove this, we could panic "for a while" but eventually this method will have to change to returning `*AccessWitnes, error`. (Or avoid the `if` check?)

Reply:
yes the if check should not be necessary, I am just setting the panic to catch a rebase issue. I'll add a TODO here.